service_test.go

```diff
@@ -383,22 +383,22 @@ func TestMakeConfig(t *testing.T) {
 		err := makeConfig(h, s)
 		So(err, ShouldBeNil)
 		So(h.Config.DHTPort, ShouldEqual, DefaultDHTPort)
-		So(h.Config.EnableMDNS, ShouldBeFalse)
+		So(h.Config.EnableMDNS, ShouldBeTrue)
 		So(h.Config.BootstrapServer, ShouldNotEqual, "")
 		So(h.Config.Loggers.App.Format, ShouldEqual, "%{color:cyan}%{message}")
 
 	})
 
 	Convey("make config should produce default config from OS env overridden values", t, func() {
 		os.Setenv("HOLOCHAINCONFIG_DHTPORT", "12345")
-		os.Setenv("HOLOCHAINCONFIG_ENABLEMDNS", "true")
+		os.Setenv("HOLOCHAINCONFIG_ENABLEMDNS", "false")
```

Comment:
Why is this being set here. I would understand this test to test makeConfig() which either sets this to true or false and which is also being tested below.

Are you testing that you can override the (new) default (true) with false?

Reply:
yes the latter